CloudinaryDotNet/ApiShared.cs

```diff
@@ -71,7 +71,7 @@ public partial class ApiShared : ISignProvider
         /// <summary>
         /// User agent for cloudinary API requests.
         /// </summary>
-        public static string USER_AGENT = BuildUserAgent();
+        public static string USER_AGENT = RuntimeInformation.FrameworkDescription;
```

Comment:
The name of this constant does not correspond the content. Very confusing. Please refactor.